Makefile

```diff
@@ -84,6 +84,16 @@ clean:
 distclean: clean
 	rm -fr cscope.*
 
+dist:
+	pkgname=pxa-impedance-matcher-$$(git describe --dirty); \
```

Comment:
I'm not 100% happy with this. It will be close to what ./genver.sh produces but it's not guaranteed to be the same. But short of switching to autotools I couldn't figure out a better way.

Reply:
Hm. This didn't work for me. But it turns out that at least for GitHub, this is also unneeded, as tags will automatically create release tarballs that can be downloaded. Did that now. Ist that sufficient for you, or do you need that to work outside of GitHub?

Reply:
> Could you expand on didn't work. If it's something simple it might be worth another try.

With a tag set, your snippet gives me

```
pkgname=pxa-impedance-matcher-$(git describe --dirty); \
rm -f $pkgname.tar.gz; \
git archive --prefix=$pkgname/ HEAD -o $pkgname.tar; \
mkdir -p $pkgname; \
./genver.sh >$pkgname/version.h; \
tar -rf $pkgname.tar $pkgname/version.h; \
rm -rf $pkgname; \
gzip $pkgname.tar
fatal: No tags can describe '1f630cb76ade923198df2fd4453020173a4113b9'.
Try --always, or create some tags.
```

What does that produce on your machine?

> There will still be a problem because version.h is generated. So it'll either get the wrong commit id if the build system uses git (a common problem for buildroot) or nothing at all when built as a tarball.

Ah, right. I totally forgot about this, sorry. Hmm. We could probably add a fallback to `genver.sh` if `.git` does not exist in the tree, and fall back to a version that has to be bumped on each release. WDYT?